pavelib/prereqs.py

```diff
@@ -354,13 +354,3 @@ def install_prereqs():
 def log_installed_python_prereqs():
     """  Logs output of pip freeze for debugging. """
     sh("pip freeze > {}".format(Env.GEN_LOG_DIR + "/pip_freeze.log"))
-
-
-def print_devstack_warning():  # lint-amnesty, pylint: disable=missing-function-docstring
```

Comment:
@feanil @salman2013: Is there a quick fix to revert this change, or should I revert the whole thing? We could also temporarily remove the call to `print_devstack_warning()`, but I'd have the question as to whether this warning should really go away? In all cases, what is going to be the quick fix since this is breaking things? Thanks.

Reply:
This PR was reverted in https://github.com/openedx/edx-platform/pull/33630 until this issue can be fixed. The changes did make it to Prod (temporarily), so other code seems to be ok.